include/itkStructurePreservingColorNormalizationFilter.h

```diff
@@ -86,24 +124,24 @@ class StructurePreservingColorNormalizationFilter : public ImageToImageFilter< T
   // approach could in theory work in other circumstances.  In that
   // case it might be better to have NumberOfStains be a template
   // parameter or a setable class member.
-  static constexpr SizeValueType NumberOfStains {2};
+  static constexpr SizeValueType NumberOfStains {2}; /**< Hematoxylin and eosin; there are two stains supported. */
   // Parameters for subroutines
-  static constexpr SizeValueType maxNumberOfIterations {0}; // For Virtanen's non-negative matrix factorization algorithm.
-  static constexpr SizeValueType maxNumberOfRows {100000}; // Select a subset of the pixels if the image has more than this
-  static constexpr CalcElementType SecondPassDistinguishersThreshold {0.90};
-  static constexpr CalcElementType BrightPercentileLevel {0.80};
-  static constexpr CalcElementType BrightPercentageLevel {0.50};
-  static constexpr CalcElementType VeryDarkPercentileLevel {0.01};
-  static constexpr CalcElementType epsilon0 {1e-2}; // a small matrix.array_inf_norm() value
-  static constexpr CalcElementType epsilon1 {1e-6}; // a very small matrix element
-  static constexpr CalcElementType epsilon2 {1e-12}; // a very small squared magnitude for a vector.
-  static constexpr CalcElementType lambda {0.00}; // For Lasso penalty.
+  static constexpr SizeValueType maxNumberOfIterations {0}; /**< For Virtanen's non-negative matrix factorization algorithm. */
```

Comment:
For Doxygen, these comments need to be on the line above the target item they are documenting.

Reply:
I will commit changes that use /** ... */ before the code rather than /**< ... */ after the code.